avocado/core/decorators.py

```diff
@@ -53,39 +53,3 @@ def wrap(*args, **kwargs):
     if func:
         return decorate(func)
     return decorate
-
-
-def skip(message=None):
-    """
-    Decorator to skip a test.
-    """
-    def decorator(function):
-        if not isinstance(function, type):
-            @wraps(function)
-            def wrapper(*args, **kwargs):
-                raise core_exceptions.TestDecoratorSkip(message)
-            function = wrapper
-        return function
-    return decorator
-
-
-def skipIf(condition, message=None):
-    """
-    Decorator to skip a test if a condition is True.
-    """
-    if condition:
-        return skip(message)
-    return _itself
-
-
-def skipUnless(condition, message=None):
-    """
-    Decorator to skip a test if a condition is False.
-    """
-    if not condition:
-        return skip(message)
-    return _itself
-
-
-def _itself(obj):
-    return obj
```

Comment:
They are not really the same. The code is very close indeed, but the unittest decorators are setting attributes in the test method to communicate the reason of the skip. We have a different architecture. That's the main reason I decided to implement our own decorators.